jubatus/core/driver/test_util.hpp

```diff
@@ -51,6 +53,21 @@ jubatus::util::lang::shared_ptr<jubatus::core::fv_converter::datum_to_fv_convert
   return converter;
 }
 
+jubatus::core::fv_converter::datum generate_random_datum(
+    jubatus::util::math::random::mtrand& rand,
+    int datum_length) {
+  jubatus::core::fv_converter::datum d;
+  std::string value;
+  value.resize(datum_length * 2 + 1);
+  for (int j = 0; j < datum_length; ++j) {
+    value[j * 2]     = (rand() % ('z' - 'a')) + 'a';
+    value[j * 2 + 1] = ' ';
+  }
+  value[datum_length*2] = '\0';
```

Comment:
Assigning `'\0'`to `std::string` should be avoided when the `std::string` object is used as string.